pg_metadata/test/test_sql.py

```diff
@@ -36,3 +37,58 @@ def test_html_template(self):
                 "SELECT pgmetadata.get_dataset_item_html_content('pgmetadata','lines','main')")
         )
         self.assertEqual("<p>Test title</p><b>Test title</b>", result[0][0])
+
+    def test_trigger_calculate_fields(self):
+        """ Test if fields are correctly calculated """
+
+        """ Test with geom column """
+        # Adding data
+        dataset_feature = {
+            'table_name': "'lines'",
+            'schema_name': "'pgmetadata'",
+            'title': "'Test title'",
+            'abstract': "'Test abstract.'",
+        }
+        self._insert(dataset_feature, 'dataset')
+
+        # Test insert
+        sql = "SELECT geometry_type, projection_authid, spatial_extent FROM pgmetadata.dataset"
+        result = self.connection.executeSql(sql)
+        self.assertEqual(['LINESTRING', 'EPSG:4326', '3.854, 3.897, 43.5786, 43.622'], result[0])
+
+        # Test update
+        sql = "UPDATE pgmetadata.dataset SET title = 'test lines title' WHERE table_name = 'lines'"
+        self.connection.executeSql(sql)
+        sql = "SELECT title, geometry_type, projection_authid, spatial_extent FROM pgmetadata.dataset"
+        result = self.connection.executeSql(sql)
+        self.assertEqual(
+            ['test lines title', 'LINESTRING', 'EPSG:4326', '3.854, 3.897, 43.5786, 43.622'],
+            result[0]
+        )
+
+        """ Test without geom column """
+        # First with removing geom column
```

Comment:
```suggestion
        # Test with removing the geom column
```

@drillinP The is a difference between `""" Docstring """` and `# Comment`.

`""" Docstring... """` which must be unique (in a function) and at the top of the function. It's designed for documentation of the code and for display outside (when running tests for instance).

` # My comment` is designed for developpers, to give some comments about a specific block of code. It won't be displayed outside of the file or outside the code editor.